src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -5,20 +5,116 @@
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private int buckets;
```

Comment:
we don't need two variables here. Your size should be equal to amount of Nodes in your HashMap.  And you should do resize depends on this amount(doesn't matter what is the placing of these nodes)

Reply:
@dmytro-hryhoruk 
Hi! size variable counts amount of elements that we put in the map; variable buckets counts number of filled buckets in the table and is used for resizing purposes.

Reply:
I can't see the place where you are using this variable

Reply:
size is used in getSize() method and buckets is used in reseze() method